runtime/common/src/lib.rs

```diff
@@ -37,6 +43,22 @@ macro_rules! production_or_benchmark {
 	}};
 }
 
+pub struct CurrencyEDs<T>(PhantomData<T>);
+impl<T> GetByKey<CurrencyId, Balance> for CurrencyEDs<T>
+where
+	T: pallet_balances::Config<Balance = Balance>
+		+ orml_asset_registry::Config<AssetId = CurrencyId, Balance = Balance>,
+{
+	fn get(currency_id: &CurrencyId) -> Balance {
+		match currency_id {
+			CurrencyId::Native => T::ExistentialDeposit::get(),
+			currency_id => orml_asset_registry::Pallet::<T>::metadata(currency_id)
+				.map(|metadata| metadata.existential_deposit)
+				.unwrap_or_default(),
```

Comment:
Such internal currencies will have a representation in the `asset_registry`? If not, yes, we can add a default other than 0 for them

Reply:
No, I think we do not register internal currencies. E.g.: staking.

Reply:
Ok, looking into it, they already have them `0` ED, but after https://github.com/centrifuge/centrifuge-chain/pull/1577/commits/f34cbdecb707edee247a57df704c35379a9f2fda from @NunoAlexandre they should have some value != 0 and such value is not in the `asset_registry`, so yes, we should add something different than default.

I'm still not sure what value they should have or how to set this from `CurrencyEDs` type (supposing in could happen with different currencies with different EDs).

Also, I still do not understand why polkadot 0.9.43 forces us to have some ED even for internal currencies as the stacking one.

cc @wischli

Reply:
@lemunozm yeah we need a special arm case for the Stacking Currencies. Those are internal so we should be safe hardcoding here.

Reply:
Ok. If all of you agree, I propose to merge this PR as it it, because ED is 0 at this point in our `main` branch, and later in `polkadot-v0.9.43` adding the corresponding value there (in a hardcode way as Nuno said) depending on what we chose for them (that I understand it's still not decided)